components/x-increment/__tests__/x-increment.test.jsx

```diff
@@ -0,0 +1,23 @@
+const { h } = require('@financial-times/x-engine');
+const { mount } = require('@financial-times/x-test-utils/enzyme');
+
+const { Increment } = require('../');
+
+describe('x-increment', () => {
+	it('should increment when clicked', async () => {
+		const subject = mount(<Increment count={1} />);
+		await subject.find('button').prop('onClick')();
```

Comment:
Can we use Enzyme's `.simulate('click')` here?

Reply:
there would be no way using that to wait for the state change. calling the `onClick` directly we can get hold of the promise that the action returns

Reply:
ah, of course 😊 

I think I need coffee.

Reply:
The number of times I've written this rubbish https://github.com/Financial-Times/o-lazy-load/blob/d18cb98615a3915b08bc0abfde6e1c920f252512/test/helpers/wait-until.js 😆